package.json

```diff
@@ -39,24 +39,24 @@
   "homepage": "https://github.com/thredup/rollbar-sourcemap-webpack-plugin#readme",
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.12.10",
-    "@babel/core": "^7.12.10",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.12.1",
-    "@babel/plugin-proposal-optional-chaining": "^7.12.1",
-    "@babel/preset-env": "^7.12.11",
+    "@babel/cli": "^7.15.4",
+    "@babel/core": "^7.15.5",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.14.5",
+    "@babel/plugin-proposal-optional-chaining": "^7.14.5",
+    "@babel/preset-env": "^7.15.6",
     "babel-eslint": "^10.1.0",
-    "babel-jest": "^26.6.3",
+    "babel-jest": "^27.2.0",
     "cross-env": "^7.0.2",
-    "eslint": "^7.26.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "husky": "^6.0.0",
-    "jest": "^26.6.3",
-    "lint-staged": "^11.0.0",
-    "nock": "^13.0.5",
-    "prettier": "^2.1.2",
+    "eslint-plugin-import": "^2.24.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "husky": "^7.0.2",
```

Comment:
We can remove `.husky/.gitignore` according to https://github.com/typicode/husky/releases

Reply:
It also reads
> Drop Node 10 support

I'm not sure if we should bundle this `husky` upgrade with `node-fetch` for this reason, or if it's fine as `husky` is part of `devDependencies` whereas `node-fetch` is a `dependencies`

Reply:
Yeah, since husky dev dep don't really need to worry about the node version requirement. The build targets node 8 but using 14 in dev.